api/client.php

```diff
@@ -25,7 +26,7 @@
 	function request($posts) {
 		$token=@$_POST['token'];
 		$ip=$_SERVER['REMOTE_ADDR'];
-		$url="http://127.0.0.1/api/jiekouapi.php?ip=$ip&token=$token";
+		$url=CAPU_JIEKOUAPI . "?ip=$ip&token=$token";
```

Comment:
Let's set up the first code convention. [1] ;-)

Please change all other similar changes to fits this convention.

https://github.com/CAPU-ENG/CAPUBBS/wiki/Convention#code-style